pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Writing Checklist
+
+If your PR involves adding or updating writing content,
+please check the following items before submitting your PR for review:
+
+- [ ] Use [kramdown link definition] syntax for all links.
+- [ ] If needed, also use [kramdown reference link] syntax.
+- [ ] Never link on the words "here", "click here", or "this document".
+Instead, use a phrase that describes what the reader will see.
+See the google developer [style guide page on links] for more information.
+- [ ] Use the present tense, and [avoid the future tense].
```

Comment:
Here's another one to add:

```
[ ] Where possible, use [active voice]. For example: 
     Passive: "There is no equivalent in Flutter."
     Active (preferred): "Flutter has no equivalent."
     (TBD: look for a better example...)
     One way to help identify passive voice is to search for "there is" or "there are" or other versions of "to be".

[active voice]: https://developers.google.com/style/voice
```

Reply:
Another one to add:

```
[ ] Avoid writing in first person. If needed, use [second person]. Better yet, rewrite to avoid either.
     First person: "In this section, we'll..."
     Second person: "In this section, you'll..."
     Even better: "In this section, learn how to..."
     When translated to other languages, even second person can be problematic.

[second person]: https://developers.google.com/style/person
```